README.md

```diff
@@ -157,5 +168,8 @@
 <h1 style="font-size: 24.5px; margin-top: -1pc; margin-left: 0pc;">History of creating the project</h1>

```

Comment:
You can structure this part into few smaller parts, because it looks like a wall of text and really hard to read it. At least separate the part about project improvement.